<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>325</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>325</width>
    <height>325</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>325</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StreamScreenClient</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../pics/streamscreen.png</normaloff>../../../pics/streamscreen.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="ipLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>IP Adresss:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>175</y>
      <width>76</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>20</y>
      <width>196</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 22pt &quot;Noto Sans&quot;;
color: rgb(42, 0, 255);</string>
    </property>
    <property name="text">
     <string>StreamScreen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 16pt &quot;Noto Sans&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Android Client</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>225</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: white;            /* Text color for contrast */
    border: none;            /* Remove border if not needed */
	border-radius: 6px;
background-color: rgb(0, 2, 131);
}
QPushButton:hover {
    background-color: blue;  /* Optional hover color */
}

QPushButton:pressed {
    background-color: navy;      /* Optional pressed color */
}</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>175</y>
      <width>21</width>
      <height>26</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::MarkdownText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>126</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;a href=&quot;https://beeralator.com&quot; style=&quot;color:#ffffff;&quot;&gt;Beeralator.com&lt;/a&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::LinksAccessibleByKeyboard|Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextBrowserInteraction|Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
    </property>
   </widget>
   <zorder>ipLineEdit</zorder>
   <zorder>label_3</zorder>
   <zorder>portLineEdit</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label</zorder>
   <zorder>imageLabel</zorder>
   <zorder>connectButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
